src/Reflection/Php/PhpFunctionReflection.php

```diff
@@ -296,19 +252,4 @@ public function acceptsNamedArguments(): TrinaryLogic
 		return TrinaryLogic::createFromBoolean($this->acceptsNamedArguments);
 	}
 
-	private function isFunctionNodeVariadic(Function_ $node): bool
```

Comment:
I wonder why this logic is no longer reflected in the patch. it seems it was dead code before

Reply:
It's really old code, maybe there just wasn't a test for it. Since we're in 2.0.x, we can risk it, and maybe introduce it back if it breaks someone's scenario.

Reply:
I think it was unnecessary logic I implemented with https://github.com/phpstan/phpstan-src/commit/0ee67d58c66bf5c603cb4ba60baa91f3c5030f33#diff-3ac4e5d4a520618a490963b70e3fcd428d56aea61683641020393167fff2ca5d

deleting the lines on 2.0.x before this PR, also doesn't break tests :).

all good.